korangar/src/graphics/renderers/picker/tile/title.wgsl

```diff
@@ -21,6 +21,6 @@ fn vs_main(
 }
 
 @fragment
-fn fs_main(@location(0) @interpolate(flat) identifier: u32) -> @location(0) u32 {
-    return identifier;
+fn fs_main(@location(0) @interpolate(flat) identifier: u32) -> @location(0) vec2<u32> {
+    return vec2<u32>(identifier, 1);
```

Comment:
I am sort of only half happy with this magic number here. Maybe we should add a comment?

Reply:
We could maybe remove this magic value by using a pipeline specialization constant? Should I try this approach? Then we could use the enum value directly with no magic constants.

Reply:
That would be super nice actaully

Reply:
I implement it later this evening.